src/components/Display.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Display.css';
+
+function Display(props) {
+  const { result } = props;
+  return (
+    <div className="display">
+      <span className="result">{ result }</span>
+    </div>
+  );
+}
```

Comment:
[Optional]

Why not use arrow function instead?

Also, we could destructure `result` directly from the argument since `props` is not used elsewhere in the function. `Display({ result })`